Resources/ServerInfo/Guidebook/Science/Science.xml

```diff
@@ -12,6 +12,8 @@ The most important thing inside your department is the R&D server, which stores
 
 Each technology costs [color=#a4885c]Research Points[/color] and unlocks recipes at lathes. Some technologies will also have prerequesites you have to unlock before you can research them.
 
+The technologies can be viewed [textlink="here" link="Technologies"].
+
```

Comment:
For accessibility reasons you should avoid linking something as "here" and it should be somewhat self-descriptive, e.g. "The technologies can be viewed on the <Technologies page>" or just "here <link>".